src/modules/blog/blog.service.ts

```diff
@@ -0,0 +1,73 @@
+import { AppDataSource } from '../../data-sourse'
+import { Blog } from '../../entities/Blog'
+import { BlogCreateDto } from './dtos'
+import { BlogUpdateDto } from './dtos/update-blog.dto'
+import { BlogParamsDto } from './schemas'
+
+export class BlogService {
+  private blogRepository = AppDataSource.getRepository(Blog)
+
+  async createBlog(payload: BlogCreateDto): Promise<Blog> {
+    const newBlog = new Blog()
+
+    newBlog.author_id = payload.author_id
```

Comment:
in previous task. You implemented authentication and authorization. You can get user from request. by its accessToken. it shouldn't come from request body. extract user from token and assign it here.